playwright/src/main/java/com/deque/html/axecore/playwright/AxeBuilder.java

```diff
@@ -474,13 +472,38 @@ private Object finishRun(ArrayList<Object> partialResults) {
     Page blankPage = browser.newPage();
     blankPage.evaluate(getAxeScript() + getAxeConfigure(hasRunPartial));
 
+    String partialResString = "";
+    try {
+      ObjectMapper mapper = new ObjectMapper();
+      partialResString = mapper.writeValueAsString(partialResults);
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+    int sizeLimit = 15_000_000;
```

Comment:
We should be consistent with the `sizeLimit` we've set with the integrations [core-npm has it set at `60_000_000`](https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/src/index.ts#L290)

```suggestion
    int sizeLimit = 60_000_000;
```